tests/requirements.in

```diff
@@ -1,10 +1,6 @@
-datadiff
+pytest<=3.10.1
```

Comment:
Is there a reason for using pytest <=3.10.1?

Reply:
Yes @arajkumar, This is the issue I was discussing with Dharmendra. For the PyTest version  >3.10.1, I am getting error **"Fixture "rdb" called directly. Fixtures are not meant to be called directly,"**, we looked into few online discussions about this, here is the link to one of them https://github.com/wolever/parameterized/issues/60 
To use grater version we will need to modify multiple test cases, so I have pointed it to 3.10.1 version.

Reply:
Ack!